app/services/twilio_message.rb

```diff
@@ -11,15 +11,7 @@ def self.activate(number)
     client.messages.create(
       from: Figaro.env.twilio_phone_number,
       to: number,
-      body: "Welcome to Kackle!"
-    )
-  end
-
-  def self.deactivate(number)
-    client.messages.create(
-      from: Figaro.env.twilio_phone_number,
-      to: number,
-      body: "Successfully unsubscribed."
+      body: "Welcome to Kackle! Respond with 'no' if you believe you received this message in error."
```

Comment:
Think that's right. If they signed up on the website they were active (so the default value on the users table was `true` for `active`), but we can swap that out.